src/transformers/models/gpt_neox/tokenization_gpt_neox_fast.py

```diff
@@ -73,9 +73,15 @@ class GPTNeoXTokenizerFast(PreTrainedTokenizerFast):
             The beginning of sequence token.
         eos_token (`str`, *optional*, defaults to `<|endoftext|>`):
             The end of sequence token.
+        pad_token (`str`, *optional*):
+            Token for padding a sequence.
         add_prefix_space (`bool`, *optional*, defaults to `False`):
             Whether or not to add an initial space to the input. This allows to treat the leading word just as any
             other word. (GPTNeoX tokenizer detect beginning of words by the preceding space).
+        add_bos_token (`bool`, *optional*, defaults to `False`):
+            Whether or not to add a `bos_token` at the start of sequences.
+        add_eos_token (`bool`, *optional*, defaults to `False`):
```

Comment:
I have switched OLMo to not add the eos token by default, since that is its behavior at inference time. Consequently the changes to this tokenizer are no longer strictly needed. There is no harm to keep them though, and the largest parts of the changes are copied from Llama anyways.

Reply:
yeah I don't think that's an issue!